tractor/_child.py

```diff
@@ -1,6 +1,43 @@
 import sys
 import trio
-import cloudpickle
+import argparse
+
+from ast import literal_eval
+
+from ._actor import Actor
+from ._entry import _trio_main
+
+
+"""This is the "bootloader" for actors started using the native trio backend.
+"""
+
+def parse_uid(arg):
+    name, uuid = literal_eval(arg)  # ensure 2 elements
+    return str(name), str(uuid)  # ensures str encoding
+
+def parse_ipaddr(arg):
+    host, port = literal_eval(arg)
+    return (str(host), int(port))
+
 
 if __name__ == "__main__":
-    trio.run(cloudpickle.load(sys.stdin.buffer))
+
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument("--uid", type=parse_uid)
+    parser.add_argument("--loglevel", type=str)
+    parser.add_argument("--parent_addr", type=parse_ipaddr)
+
+    args = parser.parse_args()
+
+    subactor = Actor(
```

Comment:
It's funny, `__init__()` almost feels kinda pointless now except for the `mp` case.